python/.gitignore

```diff
@@ -0,0 +1 @@
+GNDStk.so
```

Comment:
You should change this to *.so. GNDStk.so would be the name of the dynamic library if you compile with python 2.x, which is not what we want.

Reply:
OK, done.